WordPress/src/test/java/org/wordpress/android/ui/bloggingprompts/onboarding/BloggingPromptsOnboardingUiStateMapperTest.kt

```diff
@@ -3,72 +3,83 @@ package org.wordpress.android.ui.bloggingprompts.onboarding
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
 import org.wordpress.android.R
+import org.wordpress.android.R.string
 import org.wordpress.android.ui.avatars.TrainOfAvatarsItem.AvatarItem
 import org.wordpress.android.ui.avatars.TrainOfAvatarsItem.TrailingLabelTextItem
+import org.wordpress.android.ui.bloggingprompts.onboarding.BloggingPromptsOnboardingDialogFragment.DialogType.INFORMATION
+import org.wordpress.android.ui.bloggingprompts.onboarding.BloggingPromptsOnboardingDialogFragment.DialogType.ONBOARDING
+import org.wordpress.android.ui.bloggingprompts.onboarding.BloggingPromptsOnboardingUiState.Ready
 import org.wordpress.android.ui.utils.UiString.UiStringPluralRes
 
 class BloggingPromptsOnboardingUiStateMapperTest {
```

Comment:
Testing individual parts of ready state is definitely more precise, but the number of tests grows a lot when new variation of the state is added, so I switched to comparing whole state.